src/object/schedcontext.c

```diff
@@ -102,6 +102,12 @@ static exception_t decodeSchedContext_Bind(sched_context_t *sc, extra_caps_t ext
         return EXCEPTION_SYSCALL_ERROR;
     }
 
+    if (sc->scRefillMax == 0 || !refill_ready(sc) || !refill_sufficient(sc, 0)) {
```

Comment:
I don't think this change should be merged. Servers cannot know the state of an SC when they invoke it for operations.